dags/dim_active_users_aggregates.py

```diff
@@ -73,6 +73,7 @@
             name=f"{app_name}_{APP_NAME_POSTFIX}_dim_check",
             image=IMAGE,
             arguments=[
+                "dim",
```

Comment:
updated the dim project Dockerfile, so this change is required for the command to work.